destination.go

```diff
@@ -9,41 +24,44 @@ import (
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
+type Config struct {
+	// Personal access token.
+	Token string `json:"token" validate:"required"`
+	// Databricks server hostname
+	Host string `json:"host" validate:"required"`
+	// Databricks port
+	Port int `json:"port" validate:"required" default:"443"`
```

Comment:
we can get rid of the required validation now 👀
```suggestion
	Port int `json:"port" default:"443"`
```